templates/repo/diff/comments.tmpl

```diff
@@ -54,7 +54,7 @@
 				<span class="no-content">{{$.root.locale.Tr "repo.issues.no_content"}}</span>
 			{{end}}
 			</div>
-			<div id="comment-{{.ID}}" class="raw-content hide">{{.Content}}</div>
+			<div id="issuecomment-{{.ID}}-raw" class="raw-content hide">{{.Content}}</div>
```

Comment:
It could be, but the line immediately following has a `data-write="issuecomment-{{.ID}}-write"` attribute (which I think will also leave an inconsistency) and, on the other hand, the `data-target="{{.item.HashTag}}-raw"` attribute means either `"issuecomment-{.ID}}-raw"` or `"issue-{.ID}}-raw"` depending on the context.

Reply:
Hmm yup, it was messy already. 😂